ixofoundation/ixo-webclient
# chore(entities): customise entities styling and functionality

## Scope

https://nonaredteam.atlassian.net/browse/IXO-398
https://nonaredteam.atlassian.net/browse/IXO-462
https://nonaredteam.atlassian.net/browse/IXO-460

## Work Done

Enabled functionality to select correct entity type i.e. cells or projects on selector
New search bar styling (IXO-462)
Make project module more generic to enable use with cells
Separate control panel schema for cells

## Steps to Test

Run app
See that search bar works as expected and is styled as per figma designs
See that you can search for cells and click into a cell
